iRonCub-Mk1/iRonCub-noLegs_wbd.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <!DOCTYPE robot PUBLIC "-//YARP//DTD yarprobotinterface 3.0//EN" "http://www.yarp.it/DTD/yarprobotinterfaceV3.0.dtd">
 
-    <robot name="iRonCub-Mk1" build="1" portprefix="icub" xmlns:xi="http://www.w3.org/2001/XInclude">
+    <robot name="@YARP_ROBOT_NAME@" build="1" portprefix="icub" xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
How does this work? Is it really changing the name? I think this name is not really used, but I might be wrong

Reply:
I am not sure actually, I took a wild guess and just tried it on the robot. `yarprobotinterface` gets launched without issues! Also when switching the robot name.

So either it works, or as you said, it's not used 😆

Reply:
I would avoid to change it then

Reply:
> I would avoid to change it then

Reverted back to `iRonCub-Mk1` in https://github.com/dic-iit/robots-configuration/pull/9/commits/2dd788519db8488b4f8d074f81566c5a8038ec44.

As you said, the robot name doesn't seem to be used.